gpii/node_modules/matchMakerFramework/src/MatchMakerUtilities.js

```diff
@@ -130,11 +130,15 @@ var fluid = fluid || require("infusion"),
      * to an array as an el-path, one path for each leaf.
      *
      * @param {Object} model - The object from which to subtract leaves.
+     * @param {Boolean} ignoreArrayParsing - When a leaf value is an array, indicate whether ignore the furthur
```

Comment:
s/furthur/further .... unless you're referring to the band "Furthur", see: https://en.wikipedia.org/wiki/Furthur_(band) 😝

Reply:
Yes, let's play some music...

Fixed.